cumulusci/tasks/bulkdata/mapping_parser.py

```diff
@@ -51,7 +53,7 @@ class MappingStep(CCIDictModel):
     "Step in a load or extract process"
     sf_object: str
     table: Optional[str] = None
-    fields_: Dict[str, str] = Field(..., alias="fields")
+    fields_: Optional[Union[Dict[str, str], List[str]]] = Field({}, alias="fields")
```

Comment:
Not sure that we need the keyword Optional here if we are also supplying a default. Perhaps the default implies the Optional?

Reply:
Pydantic's documentation is a little on the thin side. I don't have a good understanding of how type constraints, defaults, and validators are ordered or interact with each other.